src/resources/user_address/user_address.route.js

```diff
@@ -12,4 +19,27 @@ router.post(
   addUserAddress,
 );
 
+router.patch(
+  '/users/:userId/address/:addressId',
```

Comment:
since a user can have more than one address, the route should indicate it. The address on the route should be plural `/users/:userId/addresses/......`